hm_pyhelper/miner_param.py

```diff
@@ -132,6 +132,9 @@ def get_gateway_mfr_command(sub_command: str) -> list:
         except (UnknownVariantException, UnknownVariantAttributeException) as e:
             LOGGER.warning(str(e) + ' Omitting --device arg.')
 
+        slot_str = f'slot={slot}'
```

Comment:
slot is being ignore for backward compatible path on L123. I am in favor of totally removing backward compatible path.

Reply:
Yes, I'm not updating backward compatible code and also think it should be removed. Feel free to open up a ticket for it.